app/javascript/article-form/components/Toolbar.jsx

```diff
@@ -10,13 +10,11 @@ export const Toolbar = ({ version, textAreaId }) => {
         version === 'v1' ? 'border-t-0' : ''
       }`}
     >
-      <div className="my-2">
-        {version === 'v1' ? (
-          <ImageUploader editorVersion={version} />
-        ) : (
-          <MarkdownToolbar textAreaId={textAreaId} />
-        )}
-      </div>
+      {version === 'v1' ? (
+        <ImageUploader editorVersion={version} />
+      ) : (
+        <MarkdownToolbar textAreaId={textAreaId} />
+      )}
```

Comment:
This reverts some of the previous changes that were meant to address scrolling issue...